src/components/FormikMultiSelect.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+const MenuProps = {
+  getContentAnchorEl: null, //TODO: remove this when material-ui gets updated to version 5
+  anchorOrigin: {
+    vertical: 'bottom',
+    horizontal: 'left',
+  },
+  PaperProps: {
+    style: {
+      maxHeight: 250,
+      width: 250,
+    },
+  },
+};
+
+function FormikMultiSelect(props) {
+  const { label, id, labelStyle, formik, options } = props;
+
+  return (
+    <>
+      <InputLabel className={labelStyle}>{label}</InputLabel>
+      <Select
+        multiple
+        fullWidth
+        id={id}
+        variant="outlined"
+        value={formik.values.requirements[id]}
+        onChange={(e) =>
+          formik.setFieldValue(
+            `requirements.${id}`,
+            e.target.value,
+            /* shouldValidate = */ false
+          )
+        }
+        MenuProps={MenuProps}>
+        {options.map((name) => (
+          <MenuItem key={name} value={name}>
+            {name}
+          </MenuItem>
```

Comment:
Optional: Have you considered using checkboxes? Or maybe letting the user decide how to display these? It seems like we can add anything here so in a later PR for something like grade level, let users check category check boxes for "All high schoolers", "All college undergraduates", "All college graduates".

Again, this is optional so maybe leave for a later PR.

Reply:
Interesting, I will keep this in mind for another PR.